Dockerfile

```diff
@@ -1,17 +1,41 @@
-FROM alpine:latest
+FROM golang:1.12.5-alpine AS build
+
+RUN apk add --no-cache \
+    gcc \
+    git \
+    musl-dev \
+    nodejs \
+    openssl \
+    postgresql-client \
```

Comment:
idk if openssl and postgresql-client are required until runtime later down in the final image. Not 💯 percent tho.

Reply:
I'm not sure either, but I believe we need it so the Go compiler can properly link the libraries. We're also not compiling an static binary right now, which would allow us to remove the dependency on those packages from the final image.
Also, I believe we could move to `postgresql-libs` package here and not include `psql` itself in the final image, but haven't tested it yet and wanted to keep changes to the multi-stage setup for now.